specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -309,18 +291,10 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/401"
         '403':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/403"
-        '404':
```

Comment:
ik denk/vind dat de 404 response hier wel moet blijven

Reply:
@fsamwel, ik neem aan dat de 404 moet blijven in geval de kadastraalonroerendezaak niet bestaat?

Reply:
Ik voeg de 404 hier weer toe.